salt/modules/http.py

```diff
@@ -35,7 +40,10 @@ def query(url, **kwargs):
         opts.update(kwargs['opts'])
         del kwargs['opts']
 
-    return salt.utils.http.query(url=url, opts=opts, **kwargs)
+    try:
+        return salt.utils.http.query(url=url, opts=opts, **kwargs)
+    except Exception as exc:
+        raise CommandExecutionError(six.text_type(exc))
```

Comment:
Pylint is failing on this line with "undefined variable six" and "undefined variable CommandExecutionError"

Reply:
Should be fixed now.